Sources/Client/Model/Channel/Channel.swift

```diff
@@ -143,6 +147,7 @@ public final class Channel: Codable {
                 lastMessageDate: Date?,
                 frozen: Bool,
                 team: String = "",
+                namingStrategy: ChannelNamingStrategy? = DefaultNamingStrategy(maxUserNames: 1),
```

Comment:
I think here should b `nil` by default, because `DefaultNamingStrategy` is for direct messaging channels.

Reply:
It leads for extra data as a direct message channel by default.